### ISSUE TITLE
WDM tries to use a windows binary`msedgedriver.exe` on Mac M1

### ISSUE THREAD

**Issue by asolntsev** [bonigarcia/webdrivermanager#812]
**Description of the problem**: 
On Mac M1, WDM downloads `edgedriver_arm64.zip` which contains Windows-specific file `msedgedriver.exe` which *doesn't work on Mac*.

**Browser and version**: Microsoft Edge 100 on **Mac M1**

**Operating system**: Mac OS

**WebDriverManager version**: 5.1.0

**WebDriverManager call**: `WebDriverManager.edge().setup();`

**WebDriverManager traces**: 

```
[main] INFO io.github.bonigarcia.wdm.WebDriverManager - Using msedgedriver 100.0.1185.29 (resolved driver for Edge 100)
[main] INFO io.github.bonigarcia.wdm.WebDriverManager - Reading https://msedgedriver.azureedge.net/?restype=container&comp=list to seek msedgedriver
[main] INFO io.github.bonigarcia.wdm.online.Downloader - Downloading https://msedgedriver.azureedge.net/100.0.1185.29/edgedriver_arm64.zip
[main] INFO io.github.bonigarcia.wdm.online.Downloader - Extracting driver from compressed file edgedriver_arm64.zip
[main] INFO io.github.bonigarcia.wdm.WebDriverManager - Exporting webdriver.edge.driver as /Users/andrei/.cache/selenium/msedgedriver/mac-arm64/100.0.1185.29/msedgedriver.exe
```

**Java error**:
```java
org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
...
Caused by: org.openqa.selenium.WebDriverException: Driver server process died prematurely.
```

NB! 
The driver would work if WDM downloaded `https://msedgewebdriverstorage.blob.core.windows.net/edgewebdriver/100.0.1155.0/edgedriver_mac64.zip` instead of `edgedriver_arm64.zip`.

**Comment by bonigarcia** [comment]
Yes, it seems this is same of #804. It is already fixed in the master branch. I plan to make a new release soon.

**Comment by bonigarcia** [comment]
This is already available in WebDriverManager 5.1.1, just released.
